accumulo_plugin/plugin-accumulo/testfiles/PelzCryptoTest.java

```diff
@@ -358,14 +436,38 @@ private void verifyKeySizeForCBC(SecureRandom sr, int sizeInBytes)
   public void testPelzKeyUtilsWrapAndUnwrap()
       throws NoSuchAlgorithmException, NoSuchProviderException {
     java.security.Key fek = PelzKeyUtils.generateKey(random, 32);
-    AccumuloConfiguration conf = getAccumuloConfig(ConfigMode.CRYPTO_ON);
-    String kekPath = conf.get(Property.INSTANCE_CRYPTO_PREFIX.getKey() + "key.uri");
+    AccumuloConfiguration conf = getAccumuloConfig(ConfigMode.CRYPTO_TABLE_ON);
+    String kekPath = conf.get(PelzCryptoService.KEY_URI_PROPERTY);
     byte[] wrapped = PelzKeyUtils.wrapKey(fek.getEncoded(), kekPath);
     assertFalse(Arrays.equals(fek.getEncoded(), wrapped));
     byte[] unwrapped = PelzKeyUtils.unwrapKey(wrapped, kekPath);
     assertTrue(Arrays.equals(unwrapped, fek.getEncoded()));
   }
+  
+  @Test
+  public void testPerTableFactory() {
+    PerTableCryptoServiceFactory factory = new PerTableCryptoServiceFactory();
+    CryptoEnvironment env = new CryptoEnvironmentImpl(TABLE, TableId.of("5"), null);
```

Comment:
Same question about "5".

Reply:
This is a copy over from the '''CryptoTest.java''' written by the Accumulo team.